uwrt_mars_rover_pid_api/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(uwrt_mars_rover_pid_api)
+
+## Find catkin macros and libraries
+find_package(catkin REQUIRED COMPONENTS
+        message_generation
+        roscpp
+        uwrt_mars_rover_msgs
+        uwrt_utils
+        )
+
+###################################
+## catkin specific configuration ##
+###################################
+## The catkin_package macro generates cmake config files for your package
+## Declare things to be passed to dependent projects
+## INCLUDE_DIRS: uncomment this if your package contains header files
+## LIBRARIES: libraries you create in this project that dependent projects also need
+## CATKIN_DEPENDS: catkin_packages dependent projects also need
+## DEPENDS: system dependencies of this project that dependent projects also need
+catkin_package(
+        INCLUDE_DIRS
+        include
+        LIBRARIES
+        ${PROJECT_NAME}
+        CATKIN_DEPENDS
+        roscpp
+        uwrt_mars_rover_msgs
+)
+
+###########
+## Build ##
+###########
+
+## Specify additional locations of header files
+## Your package locations should be listed before other locations
+include_directories(
+        include
+        ${catkin_INCLUDE_DIRS}
+)
+
+## Declare a C++ library
+add_library(${PROJECT_NAME}
+        src/${PROJECT_NAME}.cpp
+        ../uwrt_utils/src/uwrt_can.cpp
```

Comment:
This is not the proper way to add a library from another ros package. Read this: https://roboticsbackend.com/ros-include-cpp-header-from-another-package/